tests/install/openqa_webui.pm

```diff
@@ -55,10 +55,13 @@ sub install_from_git {
     systemctl restart apache2 || systemctl status --no-pager apache2
     mkdir -p /var/lib/openqa/db
     EOF
+    my $sass = script_output 'perldoc -l Mojolicious::Plugin::AssetPack::Pipe::Sass';
+    script_run q{perl -pi -wE 's/\Q = (qw(sass -s)/ = (qw(sass -s --trace)/' } . $sass;
+    script_run qq{grep "sass -s" $sass};
```

Comment:
It's bad practice to use `script_run` without checking the exit code. How about

```suggestion
    assert_script_run q{perl -pi -wE 's/\Q = (qw(sass -s)/ = (qw(sass -s --trace)/' } . $sass;
    assert_script_run qq{grep "sass -s" $sass};
```

but in general I would prefer if our code can be as simple as possible resembling what users would need to do. This is quite low-level. How about a Makefile target "prepare-assets" or something that is then called here?